providers/gitlab.js

```diff
@@ -0,0 +1,103 @@
+// gitlab.js
+// https://gitlab.com/gitlab-org/gitlab-ce/blob/master/doc/user/project/integrations/webhooks.md
+// ========
+module.exports = {
+    parse: function (req, discordPayload) {
+        var body = req.body;
+        var username = body.user_name;
+        var type = body.object_kind;
+        var ref = body.ref;
+        discordPayload.username = username + " Commit";
```

Comment:
Yeah, thanks for pointing that out. Not too much a fan of overriding the webhook configured image as the person probably set it intentionally. I am a fan of this new format though, looks great!